### PR TITLE
added devsurajrai info

### PR DESCRIPTION
**Pull request by devsurajrai**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/261"></a>

### CODE REVIEW

**Comment by devsurajrai** [comment]
updated profile pic link

File: public/data/devsurajrai.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "http://github.com/devsurajrai",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/surajrai_z",
+      "icon": "twitter"
+    }
+  ]
+}
```
Comment (Panquesito7) [inline public/data/devsurajrai.json:17]: Missing an endline.
```suggestion
}

```

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "linkfree",
-  "version": "0.12.0",
+  "version": "0.16.3",
```
Comment (Panquesito7) [inline package-lock.json:3]: Did you modify this directly or was it done automatically?

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (devsurajrai) [reply package-lock.json:3]: I did not touch anything other than adding my json file in data dir

Comment (CodingSpecies) [reply package-lock.json:3]: @Panquesito7 it seems to do this with others too.

File: public/data/devsurajrai.json (hunk 1 above)
Comment (Panquesito7) [inline public/data/devsurajrai.json:17]: Missing an endline.
```suggestion
}

```

File: public/data/devsurajrai.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Suraj Rai",
+  "bio": "Web Developer Aspirant",
+  "avatar": "https://github.com/devsurajrai.png",
+  "links": [
+    {
+      "name": "Follow me on Github",
+      "url": "http://github.com/devsurajrai",
```
Comment (Panquesito7) [inline public/data/devsurajrai.json:8]:
```suggestion
      "url": "https://github.com/devsurajrai",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**